ee/client/omnichannel/components/CannedResponse/modals/CreateCannedResponse/PreviewText.tsx

```diff
@@ -5,6 +5,7 @@ import MarkdownText from '../../../../../../../client/components/MarkdownText';
 
 const PreviewText: FC<{ text: string }> = ({ text }) => (
 	<Box
+		style={{ wordBreak: 'break-word' }}
```

Comment:
Some sites suggest break-word is not standard for Webkit and that it may not work there. MDN also says this is a deprecated property. Can we confirm if this change will work in all browsers or if we should change properties?